client/scss/components/forms/_title.scss

```
@@ -40,7 +40,21 @@
     }
 
     // Avoid calling attention to the field _unless_ it’s in one of those states.
-    &:not(:hover, :focus, :placeholder-shown, [aria-invalid='true']) {
+    &:not(
+      :hover,
+      :focus,
+      :placeholder-shown,
+      [aria-invalid='true'],
+      :has(
+        ~ :where(
+          .w-field__comment-button:hover,
+          .w-field__comment-button:focus-within,
+          .w-field__comment-button--focused,
+          .w-field__comment-button--add:hover,
+          .w-field__comment-button--reveal:hover
+        )
+      )
+    ) {
```

I'm not sure it's appropriate to use `:has()` with the very specific comment button selectors here.

I found that replacing the `:hover` and `:focus` with `:hover` and `:focus-within` on the `.w-field__input` would achieve the same result. For example:

```suggestion
    &:not(
      .w-field__input:where(:hover, :focus-within) :where(&),
      :placeholder-shown,
      [aria-invalid='true']
    ) {
```

(The `:where` for grouping `:hover` and `:focus-within` can be expanded if we feel that's too much `where`s.)

The resulting selector would still be a bit complex, though. We really only need the `:is(input, textarea, .Draftail-Editor .public-DraftEditor-content)` and not the whole parent (`&`), so something like

```suggestion
    &:not(
      .w-field__input:where(:hover, :focus-within)
        :where(input, textarea, .Draftail-Editor .public-DraftEditor-content),
      :placeholder-shown,
      [aria-invalid='true']
    ) {
```

but I'm not sure which one is better.

Thoughts @thibaudcolas?

---

Hmmmhmh. Will review asap!